test/show.jl

```diff
@@ -303,6 +303,15 @@ function f13127()
 end
 @test f13127() == "f"
 
+#test methodshow.jl functions
+@test Base.inbase(Base)
+@test Base.inbase(LinAlg)
+
+@test contains(sprint(io -> writemime(io,"text/plain",methods(Base.inbase))),"inbase(m::Module)")
+@test contains(sprint(io -> writemime(io,"text/html",methods(Base.inbase))),"inbase(m::<b>Module</b>)")
+
+@test contains(Base.url(methods(eigs).defs),"https://github.com/JuliaLang/julia/tree/$(Base.GIT_VERSION_INFO.commit)/base/linalg/arnoldi.jl#L")
```

Comment:
Oh you're right, I think the method needs a fallback for no-git builds https://github.com/JuliaLang/julia/blob/37dad15e254405ed7e6dbad9647d1a77b970854d/base/methodshow.jl#L103

Will have to go make a no-git build and test that.